src/app/containers/Footer/index.stories.jsx

```diff
@@ -1,17 +1,24 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
+import { inputProvider } from '@bbc/psammead-storybook-helpers';
+import { withKnobs } from '@storybook/addon-knobs';
+import services from '../../lib/config/services';
 import { ServiceContextProvider } from '../../contexts/ServiceContext';
 import Footer from '.';
-import services from '../../lib/config/services';
-
-const stories = storiesOf('Footer', module);
 
-Object.keys(services)
-  .filter(service => service !== 'default')
-  .forEach(service => {
-    stories.add(service, () => (
-      <ServiceContextProvider service={service}>
-        <Footer />
-      </ServiceContextProvider>
-    ));
-  });
+storiesOf('Footer', module)
+  .addDecorator(withKnobs)
+  .add(
+    'testing',
```

Comment:
I guess this label was a placeholder?

Reply:
nope, miss named there... will update shortly :)